qdrant-landing/content/documentation/cloud/gcp-marketplace.md

```diff
@@ -0,0 +1,41 @@
+---
+title: GCP Marketplace
+weight: 60
+---
+
+# Qdrant Cloud on GCP Marketplace
+
+Our [GCP Marketplace](https://console.cloud.google.com/marketplace/product/qdrant-public/qdrant)
+listing streamlines access to Qdrant for users who rely on the Google Cloud Platform for
+hosting and application development. While Qdrant's clusters run on GCP, you are using the
+Qdrant Cloud infrastructure. 
+
+## Billing
+
+You don't need a credit card to sign up for Qdrant Cloud. Instead, all billing is processed
+through the GDP Marketplace. The usage of Qdrant is added to your existing billing for GCP
+services.
+
+The payment is made through your GCP Account. Our [Billing Calculator](https://cloud.qdrant.io/calculator) can give you a better idea about costs.
+
+## How to subscribe
```

Comment:
Please add a note somewhere, mentioning that creating the subscription itself, does not cause any costs. Costs are purely usage-based and a daily overview is accessible in the accounts billing section.

Reply:
I've added a cautious sentence just above `## How to subscribe`.... cautious, just in case Google changes their policies in the future.

Reply:
That is the marketplace integration we chose in every marketplace. 

- At GCP this is called [Usage Based](https://cloud.google.com/marketplace/docs/understanding-billing#services-pricing)
- AWS calls it [Metered Usage](https://docs.aws.amazon.com/marketplace/latest/userguide/metering-for-usage.html)
- Azure also calls it [Metered Billing](https://learn.microsoft.com/en-us/partner-center/marketplace/azure-app-metered-billing)

That is not something I see as a matter of change. What _could_ be a matter of change is the share we need to give for the billing, which a cloud provider does for us. But that will most likely not affect the condition, in which a customer and its subscription is. 

Therefore, please feel free to make this more apparent. Since this is also not clear to customers one time or the other.

Reply:
> As of this writing

is misleading IMO and should not be said.

Reply:
Thanks for the links to our marketplace integrations. I've updated to point the reader to those links.

I'll leave the discussion of "As of this writing" until a later time. (I don't like using it, but now that you've pointed me to the docs that we use from our supported Cloud providers, we can point to them.)

Reply:
Thanks @pansen ! 

I think I've addressed your first two points at the end of the `## Billing` section:

```
Costs from cloud providers are based on usage. You can subscribe to Qdrant on
the GCP Marketplace without paying more.
```

I did not include your 3rd point because:
- You marked that as optional
- For me, it's a "best practice" to minimize the number of screenshots